content/docs/command-reference/remote/index.md

```diff
@@ -103,7 +103,7 @@ remote = myremote
 > [Create a Bucket](https://docs.aws.amazon.com/AmazonS3/latest/gsg/CreatingABucket.html).
 
 ```dvc
-$ dvc remote add newremote s3://mybucket/myproject
+$ dvc remote add newremote s3://mybucket/path
```

Comment:
Changed possibly confusing concepts like "project" or "key" to a more generic "path" in URLs (mostly S3 URLs had this problem).

Reply:
since we change this - we might want to use `-d` ?

Reply:
Probably not in this one under the **Customize an additional S3 remote** example (meant as a continuation of the previous example where a default local remote is added).